ENIGMAsystem/SHELL/Platforms/General/PFmain.cpp

```diff
@@ -67,6 +69,9 @@ int enigma_main(int argc, char** argv) {
   showWindow();
 
   while (!game_isending) {
+    enigma_user::window_set_caption(enigma_user::room_caption);
```

Comment:
Please throw in an `if (! enigma_user::room_caption.empty())`. A big part of this was to stop doing that every frame. Another big part was to allow disabling the behavior easily.

Reply:
It seems it requires a cast to `std::string` first since it is a var, is that ok? I think I'm the one who made it var recently so you can set it to score without needing `string(score)` or similar.

```cpp
    if (!((std::string)enigma_user::room_caption).empty())
      enigma_user::window_set_caption(enigma_user::room_caption);
```